### ISSUE TITLE
Simple Harvest ignores Fortune enchantment

### ISSUE THREAD

**Issue by Dekamir** [Real-Gecko/XPFromHarvest#13]
Drops should increase when the held item has the Fortune enchantment (tool type doesn't matter).
Instead, regular drop tables apply.

Doesn't affect regular (left-click) breaking.

**Comment by Real-Gecko** [comment]
So it works with block breaking and does not with simple harvest?

**Comment by Dekamir** [comment]
Exactly.

**Comment by Real-Gecko** [comment]
What version of MC?

**Comment by Dekamir** [comment]
1.18.1. Latest available mod for Forge.

**Comment by Real-Gecko** [comment]
I see, with block break I use `block.dropXpOnBlockBreak` in 12.2 or `block.popExperience` in later versions which I guess is triggered after block is broken and takes enchantments into account by MC itself. For simple harvest I simply use `EntityXPOrb` in 12.2, `ExperienceOrbEntity` in 16.5 or `player.giveExperiencePoints` in 1.18. Is there any way to spawn XP orb by MC that will work exactly like `popExperience`?

**Comment by Dekamir** [comment]
**I used to use a hacky way to do it.**
Check if the player is holding a tool with Fortune (and is in Survival/Adventure - or not Creative) > Spawn more ores when block is broken.

I'm not proud of it, I just couldn't find a better way to do so.

**Comment by Real-Gecko** [comment]
Well if Forge does not provide better way to do it then your solution is the one to go. I'll take a look when I have some time.

**Comment by Real-Gecko** [comment]
By the way do you know any good source of Forge API documentation? With good description of classes and methods? This one seems shallow to me https://mcforge.readthedocs.io/en/1.18.x/

**Comment by Dekamir** [comment]
I mostly use that one, too. Forge is cryptic, and there is not many documentors outside of themselves.
Maybe I'll find some other people complaining and find other ways.

_BTW, I'm talking about the item drops, not XP. I just realised I may have forgot to mention that. Just to check._

**Comment by Real-Gecko** [comment]
> BTW, I'm talking about the item drops, not XP. I just realised I may have forgot to mention that. Just to check.

Good point BTW. Please remind me do players get more XP from mining with fortune enchanted pickaxe?

**Comment by Dekamir** [comment]
No (according to Wiki).
In fact, there is no mechanic in Vanilla to increase XP gains.

**Comment by Real-Gecko** [comment]
OK, then it will only affect drops.

**Comment by Real-Gecko** [comment]
Fixed in 1.3.0
